terms/demographics/sexChromosome.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "sage.annotations-demographics.sexChromosome-0.0.1",
+  "type": "string",
```

Comment:
This should probably have the same enum as 'sex'.

Reply:
Is there any time when someone would need to record a sex chromosome abnormality here, in which case we'd want a string?

Reply:
My understanding is that the value of this field would be an actual sex chromosome, e.g. "XX", "XY".

Reply:
That is correct. For the CMC study, for example, we have a couple of XXY

Reply:
Ah, I see. It might be useful to clarify what is expected here by either: 
1. an enumerated list for 'XX', 'XY', 'XXY' (with future additions, if needed), OR
2. clarifying in the description what is expected (i.e.  "description": "Sex identified by SNP genotype (e.g. XX, XY)")

Reply:
> Ah, I see. It might be useful to clarify what is expected here by either:
> 
> 1. an enumerated list for 'XX', 'XY', 'XXY' (with future additions, if needed), OR
> 2. clarifying in the description what is expected (i.e.  "description": "Sex identified by SNP genotype (e.g. XX, XY)")

I agree